control_plane/roles/webui_awx/tasks/install_awx.yml

```diff
@@ -32,6 +32,18 @@
     dest: "{{ awx_operator_folder }}"
     version: "{{ awx_tag }}"
 
+- name: Modifying livenessprobe value
+  replace:
+    path: "{{ awx_manager_file }}"
+    regexp: 'initialDelaySeconds: 15'
```

Comment:
Why we have so many magic numbers hardcoded?

Reply:
Updated to variable